code/test-utils/src/main/java/com/adobe/marketing/mobile/services/TestDataStoreService.java

```diff
@@ -16,7 +16,7 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 
-public class FunctionalTestDataStoreService implements DataStoring {
+public class TestDataStoreService implements DataStoring {
```

Comment:
this seems to be a mock, can we rename it as MockDataStoreService?

Reply:
I think it makes sense to rename the class to MockDataStoreService in this PR as you are already renaming the class, then in your next PR continue with the refactoring. That way you are not renaming the class here and then again in your next PR. 
Same comment for TestNetworkService -> MockNetworkService